ncard_app/static/style.css

```diff
@@ -56,6 +56,10 @@
     background-image: linear-gradient(to right, #009070, #A5D2C3);
 }
 
+.background-color{
```

Comment:
`background-color` is not a good descriptive name for what this style does. Is the `.ncard-grad` class still in use? If not, you can rename `background-color` to `ncard-grad`.